app/controllers/browse_controller.rb

```diff
@@ -12,49 +10,20 @@ def index
   def show
     page = MainstreamBrowsePage.find("/browse/#{params[:top_level_slug]}")
     setup_content_item_and_navigation_helpers(page)
-
-    respond_to do |f|
-      f.html do
-        show_html(page)
-      end
-
-      f.json do
-        render(json: {
-          content_id: page.content_id,
-          navigation_page_type: "First Level Browse",
-          breadcrumbs: breadcrumb_content,
-          html: second_level_browse_pages_partial(page),
-        })
-      end
-    end
+    show_html(page)
   end
 
 private
 
   def show_html(page)
```

Comment:
As both the `show_html` and `index_html` methods have had the logic stripped out, can they be re-integrated back into the `show` and `index` methods?